packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2787,6 +2845,26 @@ export default mixins(
 	}
 }
 
+.demo-zoom-menu {
+	$--zoom-menu-margin: 5;
+
+	position: fixed;
+	left: 10px;
+	width: 200px;
+	bottom: 10px;
+	line-height: 25px;
+	color: #444;
+	padding-right: 5px;
+
+	&.expanded {
+		left: $--sidebar-expanded-width + $--zoom-menu-margin;
+	}
+
+	button {
+		border: var(--border-base);
+	}
+}
```

Comment:
seems like we are duplicating css in zoom. some of which here is unnecessary. can you reduce duplication between classes here? there's a few different approaches you can take here. for example, you can add a container to set positioning and keep zoom menu styling in a separate class